cql3/expr/expression.cc

```diff
@@ -2323,5 +2323,31 @@ bool is_single_column_restriction(const expression& e) {
 
     return result;
 }
+
+const column_value& get_the_only_column(const expression& e) {
```

Comment:
It looks deceptively similar to the previous function which only checks if the expression is single-column. So:
1. all my comments from the previous commit also apply here
2. perhaps it's worth to deduplicate and return `optional<std::reference_wrapper<column_value>>` from this function, so that you can both check if it's single column and get the value if it exists?

Reply:
:+1:, I added a function like this and now there's no duplication.